diff --git a/node_modules/@rnmapbox/maps/src/components/HeadingIndicator.tsx b/node_modules/@rnmapbox/maps/src/components/HeadingIndicator.tsx
index 03cd306..b5ba9c9 100644
--- a/node_modules/@rnmapbox/maps/src/components/HeadingIndicator.tsx
+++ b/node_modules/@rnmapbox/maps/src/components/HeadingIndicator.tsx
@@ -5,6 +5,7 @@ import { BaseProps } from '../types/BaseProps';
 
 import { SymbolLayer } from './SymbolLayer';
 import Images from './Images';
+import { View } from 'react-native';
 
 const style = {
   iconImage: 'userLocationHeading',
@@ -19,7 +20,7 @@ type Props = BaseProps & {
 
 const HeadingIndicator = ({ heading }: Props) => {
   return (
-    <React.Fragment key="mapboxUserLocationHeadingIndicatorWrapper">
+    <View key="mapboxUserLocationHeadingIndicatorWrapper">
       <Images
         images={{ userLocationHeading: headingIcon }}
         key="mapboxUserLocationHeadingImages"
@@ -34,7 +35,7 @@ const HeadingIndicator = ({ heading }: Props) => {
           ...style,
         }}
       />
-    </React.Fragment>
+    </View>
   );
 };
 
